### PR TITLE
feat: add blog post about collaborative work

### PR DESCRIPTION
**Pull request by jesusreal**
Special thanks to @fischeversenker for his feedback about the text 🙏🏼

### What has been done
- Added a blog post about collaborative work
- Link to blog post [here](https://satellytescommain21751-blogpostjrthoughtsoncollaborat.gtsb.io/blog/thoughts-on-collaborative-work/)

### Things to consider
- I plan to merge this pr at the end of the February, or after a period of around 10 days after the last blogpost, to have their publishing better distributed in time.

### CODE REVIEW

File: blog-posts/thoughs-on-collaborative-work.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+leadboxText: Are you excited about getting the most out your team work? Would you like to work with great professionals? Then join us!
+
+
+---
+
+Most of us developers, despite being more or less aware of the benefits of doing more work together, work on our own most of the time. Why? What can we do about it? Let's dive into it!
+
+## Is it just about programming?
+
+Short answer, no, it’s not. As software developers we don’t just write code, we also play a crucial role in actively shaping, challenging and clarifying requirements. And this happens in many different flavours: emails, issue tracking systems, chat tools, team meetings, inter-team meetings... Every area of our work can benefit if we approach it in a more collaborative way.
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: That felt abrupt but I like the pivot towards requirements.

File: blog-posts/thoughs-on-collaborative-work.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## The good, the bad and the ugly. Or is it just good?
+
+What is the greatest hesitance when thinking about working more with other colleagues? Well, most of us think it is generally a waste of time, that it is an inefficient way to work, that there is no time for that. But if we dig deeper, we will find two strong causes. First, what I call "the developer pride". We think we are great if we solve problems on our own, if we do it without help, but if we ask for help it means we are not good enough. Second, shame. Often times we feel like what we do is not good enough to be shared. The idea of someone watching you work, the typos you do, the uncertainty with which you approach problems, etc. is frightening for most of us. Literally “stage fright”.
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: > most of us think it is generally a waste of time

That's a harsh assumption 😳 I would rather start explaining that it may feel cumbersome? Anything that sounds friendlier than that.

File: blog-posts/thoughs-on-collaborative-work.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+These fears are common both in junior and in senior developers, but the truth is that we are all learning all the time, and the more we interact with others, the more we learn. I am not an exception, but I decided to open myself more to collaborative work for three main reasons:
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: `junior` can be replaced with immature and senior with mature as `junior` which is often preferable as `junior` implies kind of a downward view.

File: blog-posts/thoughs-on-collaborative-work.md (hunk 2 above)
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: "developer pride" & shame.

I would have expected to read about the common imposter syndrome too. It somehow lays in between?

Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: >  We think we are great if we solve problems on our own,

You could describe that as "we all have a good feeling when doing things on our own" instead of implying actual thoughts about being better or worse. I think it's often not such a direct feeling.

Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: > stage fright

I like that but this is mostly related to pair programming or what scenario do you have in mind ?

File: blog-posts/thoughs-on-collaborative-work.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+- I was not happy with the amount and quality of collaborative work in my teams,
+- I consider these beliefs mostly as prejudices, and
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: make it a clear list, no need for coupling words.

```suggestion
- I consider these beliefs mostly as prejudices
```

File: blog-posts/thoughs-on-collaborative-work.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+- Every time I was approaching work in a more collaborative way, even though feeling unnatural, it showed to be quite powerful.
+
+Don't get me wrong, I am not talking about overdoing it. It is important to keep the balance, working many hours per day in groups is much more tiring, challenging and also more inefficient than just working on your own. So what we really want is to find the range where collaborative work produces a positive impact in our working life. In order for that to happen, it is crucial to accumulate experiences and get a feeling of which collaboration paradigm is better to use for each use case. Daily meeting? Knowledge sharing session? Second opinion? Think of a toolbox for team collaboration, this is exactly what is offered to you. Every person, every situation, every topic is different, that's what makes this journey exciting and why you want to have a variety of options and adapt each time.
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: I like the toolbox thought here and the way you sell the idea to pursue collaboration work.

File: blog-posts/thoughs-on-collaborative-work.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+---
+
+Most of us developers, despite being more or less aware of the benefits of doing more work together, work on our own most of the time. Why? What can we do about it? Let's dive into it!
+
+## Is it just about programming?
+
+Short answer, no, it’s not. As software developers we don’t just write code, we also play a crucial role in actively shaping, challenging and clarifying requirements. And this happens in many different flavours: emails, issue tracking systems, chat tools, team meetings, inter-team meetings... Every area of our work can benefit if we approach it in a more collaborative way.
+
+## The good, the bad and the ugly. Or is it just good?
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: I love the western but I don't get the headline in those aspects. Maybe less abstract?

File: blog-posts/thoughs-on-collaborative-work.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+There are many opportunities to collaborate with a colleague while working on a task, and it doesn't need to be monotone or boring, or useless. It can be fun, productive, fulfilling, and yes, challenging if you are not used to do it, but meaningful and definitely worthy in any case. Nevertheless we keep working way too much in silos. Most of the time we use daily meetings or other team meetings, together with written communication (issue tracker system, pull Requests, chat tool) to grab feedback on our work or to discuss topics. They do not fit well all purposes though. 
+
+## In practice 
+
+As you will realise in this section, the common factors of any kind of collaborative work are: clear goals, preparation, focus, constantly challenging yourself and each other’s thoughts and a constructive attitude. 
+
+There are many situations that greatly benefit from doing tasks in small groups, here I am going to describe the ones that more resonate with me and that I imagine to be more valuable for you. Don't limit yourself to this list, but use it also as inspiration to find your own ways.
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: pick me up. The headline tells me that it's going to be about how to start, but the text only talks about in a meta way about the following content without taking my hand and telling me what's up next

File: blog-posts/thoughs-on-collaborative-work.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+#### The powerful start 🚀
+
+When you start working on a task, sometimes it feels intimidating. If you are lacking knowledge and you can find resources on the Internet, that’s the way to go. But if the overwhelming complexity lies in project specifics, or you just have a colleague that is really proficient in that area, the best you can do is to ask for a short meeting. 15 to 30 minutes should suffice, keep it short and don’t enter into too many details. Depending on the situation, this session is either about  your colleague sharing their knowledge with you or you both trying to figure out how to tackle the task. In addition, if you both agree and the schedule allows it, you might decide to have an ad-hoc session of pair programming. 
+
+I have to admit, most of the time I am shy to ask for these sessions, and after having one of them I always feel amazed about how helpful it was and how much progress we did, and I always regret not doing it more often. Do not hesitate, ask for those sessions.
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: 🙌

File: blog-posts/thoughs-on-collaborative-work.md (hunk 7 above)
Comment (georgiee) [reply blog-posts/thoughs-on-collaborative-work.md]: After reading the next sections I would have hoped that you tell me "hey I will present you four amazing ways to collaborate with your team" and such.


The following headlines with emojis feel and look great but it's difficult to know what in them. For example "The doubtful solution " is actually about "the second opinion". I think you should be more clear about the content instead of creating creative headline to make it more accessible for people?

File: blog-posts/thoughs-on-collaborative-work.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+Phew, that’s really bad. What can we do? Nominate 2 (max. 3) team members to attend the meeting and have a preparation meeting to prioritise the topics that need to be discussed, to define what is the right information to provide and the right questions to ask for each of the topics, and to update the meeting agenda if necessary. Consider always the time available for the actual meeting while preparing, since this influences the amount of items you can clarify and how deep you can discuss them.
+
+## Conclusion
+
+I hope after reading this article you feel inspired and empowered to improve some work practices that are not working quite well in your team at this time.
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md:77]: The "In practice gave" me that good felling indeed!

File: blog-posts/thoughs-on-collaborative-work.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+
+## Conclusion
+
+I hope after reading this article you feel inspired and empowered to improve some work practices that are not working quite well in your team at this time. 
+
+Your initial reaction to working in a more collaborative way might be rejection, since it can be unpleasant to do something that exposes yourself to others, mostly if you are not used to doing so in a working environment. But there are so many positive consequences, that I firmly believe we should conquer those hesitances. You will be working as part of a team that communicates well, a team that is flexible and strong to overcome any challenge it will face, a team where every one grows and takes ownership, and definitely a team where everyone is engaged and happy. And the cherry on the cake is that you will deliver a much better product. Isn't that fantastic? 🎉
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md:79]: That's a good conclusion🙏

**Review by georgiee — CHANGES_REQUESTED** [review]
Hey thanks! That was a good reading.

I really liked part 2 where as the first part 2 where you try to approach the topic feels somehow out of the place. 

I think the blog post would already work if you focus on the four collaboration strategies and frame it as "the four best ways to start collaborating" with a much smaller motivation/introduction for the topic.

If you really want to keep the first parts I think you need a rewrite, maybe a rewrite under the premise that you have a great list of four ways to start contribution and  that it's difficult for people to leave their comfort zone.

The first parst feels too pseudo scientific, abstract and too much theory that is of no use for the later part (which I could consume directly jumping into if I want)

File: blog-posts/thoughs-on-collaborative-work.md (hunk 3 above)
Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md]: I didn't use the wording you proposed but I rephrased it.

File: blog-posts/thoughs-on-collaborative-work.md (hunk 2 above)
Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md]: I like it. I adapted the sentence

Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md]: I agree. I softened it and removed the wording "Pride" and "Shame".

File: blog-posts/thoughs-on-collaborative-work.md (hunk 6 above)
Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md]: I like it, I would like to keep it as it is.

File: blog-posts/thoughs-on-collaborative-work.md (hunk 9 above)
Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md:77]: Glad to hear that! 🙌

File: blog-posts/thoughs-on-collaborative-work.md (hunk 2 above)
Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md]: It could be many flavours, basically in any of the 4 approaches  I described below. The examples before that sentence are mostly about pair-programming, I assume that is the reason for your comment. I will adapt that.

File: blog-posts/thoughs-on-collaborative-work.md (hunk 7 above)
Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md]: For the first part I agree, it is not introducing you to what it is coming. I added a sentence similar to the one you mentioned and removed some bla bla.

Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md]: For the second part, I would prefer to not do any change on the headlines. For the first 2, by reading the first sentence the reader already know what is it about. I like the creative part of it, I don't consider it misleading, and I think it triggers the curiosity of the reader. For the latter 2, I think the title is quite self-explanatory.

File: blog-posts/thoughs-on-collaborative-work.md (hunk 2 above)
Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md]: Yes, it does, they are two aspects of impostor syndrome. Thanks for pointing that out, I added a reference to it. Since you mentioned `pride` & `shame`, as I stated in another comment I removed those 2 words to soften the message.

File: blog-posts/thoughs-on-collaborative-work.md (hunk 4 above)
Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md]: Applied in a separate commit.

**Comment by jesusreal** [comment]
Thanks for sharing those great insights about this article @georgiee! 

After reading your feedback and reading the article again, I agree the first section had too much content, kind of repetitive. I have shorten them heavily, and kept what I think it is important and relevant to mention in this article. I think I just was in love with the writing of the introduction, but I didn't like it that much after a couple of weeks without actively writing it.

Apart from that I went through all your comments and answered them, and applied changes where I agreed on your feedback.

I really like how it looks now. Looking forward to publish this piece! 

Please have another look. Thanks! 🙌

File: blog-posts/thoughs-on-collaborative-work.md (hunk 2 above)
Comment (georgiee) [reply blog-posts/thoughs-on-collaborative-work.md]: It's still `most of us ` can you tone down with `For some folks` or similar because I'm sure working together involves different scales of fraction.

File: blog-posts/thoughs-on-collaborative-work.md (hunk 3 above)
Comment (georgiee) [reply blog-posts/thoughs-on-collaborative-work.md]: I saw 
>  degree of expertise, 

thanks.

File: blog-posts/thoughs-on-collaborative-work.md (hunk 6 above)
Comment (georgiee) [reply blog-posts/thoughs-on-collaborative-work.md]: I like the new introduction revolving around impostor but I must say I still don't get the headline. What are the three aspects you refer to ? Do you mean the bullet list included?

File: blog-posts/thoughs-on-collaborative-work.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+- Every time I was approaching work in a more collaborative way, even though feeling unnatural, it showed to be quite powerful
+
+Don't get me wrong, I am not talking about overdoing it, but about finding the balance, the sweet spot where collaborative work produces a positive impact in our working life. In order for that to happen, it is crucial to accumulate experiences and get a feeling of which collaboration paradigm is better to use for each use case. Daily meeting? Knowledge sharing session? Second opinion? Think of a toolbox for team collaboration, this is exactly what is offered to you. Every person, every situation, every topic is different, that's what makes this journey exciting and why you want to have a variety of options and adapt each time.
+
+## In practice 
+
+In this section I will present you four fantastic, experience-proved ways to foster collaboration in your team. As you will realise, the common factors of any kind of collaborative work are: clear goals, preparation, focus, constantly challenging yourself and each other’s thoughts and a constructive attitude. 
+
+Don't limit yourself to this list, but use it also as inspiration to find your own ways!
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: Yeah I like the flow of your text in this section after the rewrite ✨

File: blog-posts/thoughs-on-collaborative-work.md [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+
+#### The powerful start 🚀
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: h2 changes to h4. I can't find my previous comment right now but I'm sure I commented on that. That's a wrong outline and this must be h3

File: blog-posts/thoughs-on-collaborative-work.md [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+path: "/blog/thoughts-on-collaborative-work/"
+date: "2022-02-28"
+title: "Thoughts on collaborative work"
+author: Jesús Real
+authorSummary: Senior Frontend Engineer @ Satellytes
+featuredImage: images/thoughs-on-collaborative-work/group-table.jpg
+attribution:
+    creator: Marvin Meyer
+    source: https://unsplash.com/photos/SYTO3xs06fU
+seoMetaText: This article aims to offer an overview about the common beliefs around collaborative work, highlight the (many) (surprising) positive sides of it, and depicts some on-field examples that will either suit you or inspire you to find your own ways on approaching collaborative work.
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: Please shorten, it's too long for SEO. Check via https://metatags.io/ where you can see the text ellipsis:
Screenshot 2022-02-28 at 11 43 39



The images are not loading in the branch preview it seems, that's not part of this PR though.

File: blog-posts/thoughs-on-collaborative-work.md [hunk 14]
```diff
@@ -0,0 +1,79 @@
...
+date: "2022-02-28"
+title: "Thoughts on collaborative work"
+author: Jesús Real
+authorSummary: Senior Frontend Engineer @ Satellytes
+featuredImage: images/thoughs-on-collaborative-work/group-table.jpg
+attribution:
+    creator: Marvin Meyer
+    source: https://unsplash.com/photos/SYTO3xs06fU
+seoMetaText: This article aims to offer an overview about the common beliefs around collaborative work, highlight the (many) (surprising) positive sides of it, and depicts some on-field examples that will either suit you or inspire you to find your own ways on approaching collaborative work. 
+teaserText: This article aims to offer an overview about the common beliefs around collaborative work, highlight the (many) (surprising) positive sides of it, and depicts some on-field examples that will either suit you or inspire you to find your own ways on approaching collaborative work.
```
Comment (georgiee) [inline blog-posts/thoughs-on-collaborative-work.md]: > highlight the (many) (surprising) positive sides 

That feels super odd and the text would work like that too:

>  highlight the many surprising positive sides

File: blog-posts/thoughs-on-collaborative-work.md (hunk 6 above)
Comment (georgiee) [reply blog-posts/thoughs-on-collaborative-work.md]: I have commented outside of the review already. Please help here. I guess I'm not the only one with problems to understand this headline? Maybe ask colleagues? I might be wrong here.

**Review by georgiee — CHANGES_REQUESTED** [review]
1. Please fix your headlines
2. The good/bad/ugly headline, maybe you can check other people understand it, because it really doesn't ring a bell for me, even with thinking about it a lot.
3. Check the seo text to shorten/rephrase it

Aside from that, we are almost ready to merge. Thank you!

File: blog-posts/thoughs-on-collaborative-work.md (hunk 6 above)
Comment (fischeversenker) [reply blog-posts/thoughs-on-collaborative-work.md]: "The good, the bad and the ugly" is perfectly fine. It's a very well known movie title and used all over the place, over and over again. Even if you don't know the movie, the headline still makes sense. There are good bits, bad bits and ugly bits about collaboration. I don't understand the confusion.
I know the movie and I have seen the headline being used in newspapers, blog posts, etc..
Approval from my side.

EDIT: We all know [the main theme](https://www.youtube.com/watch?v=h1PfrmCGFnk) 🤠

**Comment by fischeversenker** [comment]
> The good/bad/ugly headline, maybe you can check other people understand it, because it really doesn't ring a bell for me, even with thinking about it a lot.

The headline is absolutely fine in my opinion. See [my comment](https://github.com/satellytes/satellytes.com/pull/414#discussion_r816519025).

File: blog-posts/thoughs-on-collaborative-work.md (hunk 6 above)
Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md]: FYI in the meantime I updated it to "The good, the bad and the ugly about collaboration", instead of "The good, the bad and the ugly. Or is it just good?"

File: blog-posts/thoughs-on-collaborative-work.md (hunk 12 above)
Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md]: I also didn't see that comment. I updated the headlines, thanks for noticing.

File: blog-posts/thoughs-on-collaborative-work.md (hunk 13 above)
Comment (jesusreal) [reply blog-posts/thoughs-on-collaborative-work.md]: That's great feedback, thanks for looking into that 🙏 
Updated, see image below
Screenshot 2022-03-01 at 09 53 17

**Comment by jesusreal** [comment]
Thanks for this great new round of feedback @georgiee! I applied your comments, taking care of the 3 points you stressed on your last review comment.
Apart from that, I got an external review session last evening (from my girlfriend ❤️ )  and applied some changes on punctuation, wording, and most importantly I updated the title of the article.
In you are interested about those changes, you can find them in commit https://github.com/satellytes/satellytes.com/pull/414/commits/43abe84f7e9e02e17c8535121c3a7eedd751b309.

**Review by georgiee — APPROVED** [review]
Thank you for your patience and changes, I'm super happy with the blog post and it's read to get merged 🤩 Well done 🙌
